### PR TITLE
Improve visibility of task upload time for students

### PR DESCRIPTION
**Pull request by kryonix**
Follow up of https://github.com/infomark-org/infomark-ui/pull/13. This makes the upload time more visible for students.

image

### CODE REVIEW

**Comment by kryonix** [comment]
Requires https://github.com/infomark-org/infomark/pull/10 now. Adds indication how many points students can expect. Status of grading by tutors was not clear before.

image

File: src/Pages/CourseDetail.elm [hunk 1]
```diff
@@ -1477,6 +1477,13 @@ filterExam exams enrollments =
     List.map (\( use, id, exam ) -> ( id, exam ))
         (List.filter (\( use, id, exam ) -> use) examSuccess)
 
+type alias MaybePoints =
```
Comment (vork) [inline src/Pages/CourseDetail.elm]: Why is it called `MaybePoints`? As Elm uses `Maybe` for optionals I would've expected a second `Nothing` variant. Also we have the same definition in `src/Pages/SheetDetail.elm`. Can we maybe add it to `src/Types.elm` or some other place where datatype are collected?

**Review by vork — CHANGES_REQUESTED** [review]
See my comment regarding `MaybePoints`. Otherwise it looks good.

File: src/Pages/CourseDetail.elm (hunk 1 above)
Comment (kryonix) [reply src/Pages/CourseDetail.elm]: Sounds good to me. I renamed it to `SheetPointsAchievable` and moved it to `src/Types.elm`.

**Review by vork — APPROVED** [review]
Everything looks good now

**Pull request merged**